packages/amplify_core_plugin_interface/lib/amplify_core_plugin_interface.dart

```diff
@@ -60,6 +60,8 @@ abstract class Core extends PlatformInterface {
   final AnalyticsCategory Analytics = AnalyticsCategory();
   final StorageCategory Storage = StorageCategory();
   final DataStoreCategory DataStore = DataStoreCategory();
+  
+  Map<String, StreamController> hub;
```

Comment:
The _map_ can be `final`, even if its contents will be mutable, no?

Reply:
Good call - done.